primer/brand
# Fix bug with '0' showing while VideoPlayer renders

## Summary

While the VideoPlayer is rendering a _0_ can be seen in the bottom left of the player. This PR fixes that bug.

## Steps to test:

1. Load the VideoPlayer in Storybook
2. Check that the 0 doesn't show in the bottom left during the initial render


## Contributor checklist:


## Reviewer checklist:


## Screenshots:

See the _0_ in the bottom-left of the _Before_ screenshot

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![image](https://github.com/user-attachments/assets/95ce0805-a804-423b-9e82-455d14474118)


 </td>
<td valign="top">

![image](https://github.com/user-attachments/assets/d7c36733-2619-42fd-8d44-fe17d3023b1e)

</td>
</tr>
</table>
